system.go

```diff
@@ -1,28 +1,38 @@
 package ecs
 
-// System is an interface which implements an ECS-System. A System
-// should iterate over its Entities on `Update`, in any way suitable
-// for the current implementation.
+// A System implements logic for processing entities possessing components of
```

Comment:
> Is this legit godoc?

Do you refer to the sentence starting with an article? It is valid.

References:

From https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences
>  ```go
// A Request represents a request to run a command.
type Request struct { ...
```

From https://github.com/golang/lint/blob/master/lint.go#L782
> ```go
	s := doc.Text()
	articles := [...]string{"A", "An", "The"}
	for _, a := range articles {
		if strings.HasPrefix(s, a+" ") {
			s = s[len(a)+1:]
			break
		}
	}
```